CHANGELOG.md

```diff
@@ -887,46 +893,46 @@ export default Vue.extend({
 
 ```typescript
 import {createDomain} from 'effector'
-import {fork, hydrate, serialize, allSettled} from 'effector/fork'
+import {fork, allSettled} from 'effector/fork'
 
 //app
 const app = createDomain()
+
 const fetchFriendsFx = app.createEffect<{limit: number}, string[]>({
   async handler({limit}) {
     /* some client-side data fetching */
     return []
-  },
+  }
 })
-const $user = app.createStore('guest')
-const $friends = app
-  .createStore([])
+const user = app.createStore('guest')
```

Comment:
We are replacing inconsistent store names like `user` with `$user`, aren't we? 🤔

Reply:
```suggestion
const $user = app.createStore('guest')
```